core/src/blocks/curl.rs

```diff
@@ -147,15 +149,22 @@ impl Block for Curl {
         let request = HttpRequest::new(
             self.method.as_str(),
             url.as_str(),
-            headers_value,
-            body_value,
+            headers_result["value"].clone(),
+            body_result["value"].clone(),
         )?;
 
         let response = request
             .execute_with_cache(env.project.clone(), env.store.clone(), use_cache)
             .await?;
-
-        Ok(json!(response))
+        Ok(BlockResult {
+            value: json!(response),
+            // if there are meta logs, then concatenate them and return them
+            meta: Some(Value::String(
+                headers_result["logs"].as_str().unwrap().to_owned()
```

Comment:
While I test things, I think we should make sure to have `''` as a result here if there is no log in either of executions. Mind changing it?

Reply:
`script.rs` (where the logs are created) initializes it as a `''`. If you think it'd be better with an `unwrap_or('')` I could also do that though.

Reply:
I think it's fine to have Scripts.rs return `''` and have as a convention that if logs are possible then they are set but possibly to the empty string.

But here with your concatenation `''` and `''` gets turned into `'\n'` right?